src/axolotl/utils/data/pretraining.py

```diff
@@ -196,7 +199,12 @@ def wrap_pretraining_dataset(
         # set this to 1 so downstream data_loader doesn't try to increase the batch again
         cfg.micro_batch_size = 1
     else:
-        encode = functools.partial(encode_pretraining, tokenizer, max_tokens)
+        encode = functools.partial(
+            encode_pretraining,
+            tokenizer,
+            max_tokens,
+            text_column=cfg.pretraining_dataset[0].text_column,
```

Comment:
```suggestion
            text_column=cfg.pretraining_dataset[0].text_column or "text",
```
looks like the e2e needs this?